pdf/src/campPrint/summary/SummaryDay.vue

```diff
@@ -47,23 +50,29 @@ export default {
     entries() {
       return this.scheduleEntries.map((scheduleEntry) => ({
         scheduleEntry,
-        storyChapters: this.period
+        contentNodes: this.period
           .contentNodes()
           .items.filter(
             (contentNode) =>
-              contentNode.contentTypeName === 'Storycontext' &&
+              contentNode.contentTypeName === this.contentType &&
               contentNode.root()._meta.self ===
-                scheduleEntry.activity().rootContentNode()._meta.self &&
-              !isEmptyHtml(contentNode.data.html)
+              scheduleEntry.activity().rootContentNode()._meta.self &&
+              !isEmptyHtml(contentNode.data.html) &&
+              (!this.filter || contentNode.instanceName
+                  ?.toLowerCase()
```

Comment:
How about toLocaleLowerCase? Or even slugify or any other transformation which will iron out some typos and umlauts and whatnot? (if you don't agree, just ignore this comment)

Reply:
I think this is enough for the moment.